src/Symfony/Component/EventDispatcher/EventDispatcher.php

```diff
@@ -80,7 +80,7 @@ public function getListeners($eventName = null)
      */
     public function hasListeners($eventName = null)
     {
-        return (bool) count($this->getListeners($eventName));
+        return (bool) $this->getListeners($eventName);
```

Comment:
Note that this can't happen really: that'd be a BC break.
In any case, there is a test case on this method, so this won't happen silently.